### ISSUE TITLE
Add ESLint to the backend project

### ISSUE THREAD

**Issue by aerabi** [aerabi/events#8]
We should add ESLint to the backend project and enforce the recommended rules. E.g. the `.eslintrc.js` file should contain the following content.

```js
module.exports = {
  parser: '@typescript-eslint/parser',
  parserOptions: {
    ecmaVersion: 2020,
    sourceType: 'module',
  },
  extends: ['plugin:@typescript-eslint/recommended', 'prettier', 'plugin:prettier/recommended'],
  rules: {},
};
```

You can use [this repo](https://github.com/rxjsx/rxjsx) as a reference for the `prettier` rules, `eslint` rules, and the packages to be installed.

We should add the following NPM commands to the backend project:
- `npm run lint`: Fix the fixable lint issues (including prettier)
- `npm run lint:test`: Just check if the code is well-formatted (used in the CI)

**Comment by himanshukandpal-28** [comment]
Hey @aerabi shall I pick up this issue?

**Comment by himanshukandpal-28** [comment]
Hey @aerabi added the changes.

**Comment by aerabi** [comment]
Fixed through #19.
